crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -146,6 +146,7 @@ impl Name {
         let mut namespace = if let Some(index) = attribute_find_path(attrs, &["namespace"]) {
             Some(expr_to_string(
                 &attrs[index].meta.require_name_value()?.value,
+                // requires non string literal so might be uncatchable
```

Comment:
unreachable?

Reply:
Not sure but I think so?

Reply:
What if you do like `#[namespace = 1]` then it's not a string literal?

Reply:
(as it should be `#[namespace = "1"]`

Reply:
Added a test for this